index.html

```diff
@@ -15,16 +15,20 @@
   <body>
     <header class="container">
       <h1><a href="#">Bekjon</a></h1>
-      <nav class="hamburger">
+      <div class="hamburger">
         <div class="line"></div>
         <div class="line"></div>
         <div class="line"></div>
+      </div>
```

Comment:
- [ ] According to the project requirement you should stick to the design as much as possible. So please use this `    background: rgba(103, 121, 142, 1);` to `X` buttons so it matches the design
### Your design
![your](https://user-images.githubusercontent.com/81727988/217499130-b1c653cf-d309-45fd-b9b7-42def1a07045.png)

### Fimga design
![figma](https://user-images.githubusercontent.com/81727988/217499156-5d1e18d4-a92f-448c-9e60-6c0cca0b56f9.png)